lib/errors/aggregate-error.ts

```diff
@@ -3,10 +3,11 @@ import BaseError from './base-error';
 /**
  * A wrapper for multiple Errors
  *
- * @param errors Array of errors
+ * @param errors The aggregated errors that occurred
  */
 class AggregateError extends BaseError {
-  errors: Array<AggregateError | Error>;
+  /** the aggregated errors that occurred */
```

Comment:
Doubled comment? (line 6)

Reply:
The one on line 6 is for the parameter of the constructor, this one is for the field :)